### PR TITLE
Feature/global search

### PR DESCRIPTION
**Pull request by MuhammedToichubai**
Check the code. If there are any mistakes, I will try to fix them.

### CODE REVIEW

File: src/main/java/kg/airbnb/airbnb/services/impl/AnnouncementServiceImpl.java [hunk 1]
```diff
@@ -281,20 +284,59 @@ public kg.airbnb.airbnb.dto.responses.SimpleResponse rejectAnnouncement(Long id,
...
+        } else {
             throw new ForbiddenException("Only admin can access this page!");
         }
 
     }
 
+    @Override
+    public List<AnnouncementSearchResponse> getSearchAnnouncements(Integer pageNo, Integer pageSize, String keyword) {
+
+        Pageable pageable = PageRequest.of(pageNo - 1, pageSize);
```
Comment (beksultancs) [inline src/main/java/kg/airbnb/airbnb/services/impl/AnnouncementServiceImpl.java:304]: if keyword is not null then pageable won't work?

**Review by beksultancs — COMMENTED** [review]
RFC!

**Comment by MuhammedToichubai** [comment]
conflict  and code fixed

**Comment by MuhammedToichubai** [comment]
Teacher, please check the query

**Review by beksultancs — APPROVED** [review]
LGTM!

**Pull request merged**